utils.py

```diff
@@ -64,6 +64,10 @@ def populate_parser(parser):
                         help="UNSAFE! Auto accept fingerprint during certificate verification.")
     parser.add_argument("-t", "--session-timeout", required=False,
                         help="Session expiration timeout in seconds.")
+    parser.add_argument("--force", required=False, default=False, action="store_true",

```

Comment:
There is also fingerprint confirmation message as part of Python SDK, Look for unsafe_auto_accept in APIClientArgs
Add warning in flag description so user be aware it's override --unsafe-auto-accept flag.

Reply:
Added